Boon.toml

```
@@ -58,8 +58,8 @@ ignore_list = [
     ".exe$",
 
     # Boon-specific files and directories
-    "Boon.toml",
-    "release",
+    "Boon.toml$",
```

Should this also include the start of the line, so it doesn't match something like `something-else-Boon.toml`?
 The line below does, so just curious if it would be better to make this consistent.
```suggestion
    "^Boon.toml$",
```

---

This is a very good point.